src/MoBi.Core/Commands/UpdateSimulationCommand.cs

```diff
@@ -73,6 +73,14 @@ protected override void ExecuteWith(IMoBiContext context)
          context.PublishEvent(new SimulationReloadEvent(_simulationToUpdate));
       }
 
+      private void updateReferencesToSimulation(IMoBiContext context)
+      {
+         var simulationReferenceUpdater = context.Resolve<ISimulationReferenceUpdater>();
+         simulationReferenceUpdater.SwapSimulationInParameterAnalysables(_simulationToUpdate, _simulationToUpdate);
+
+         var simulationParameterOriginIdUpdater = context.Resolve<ISimulationParameterOriginIdUpdater>();
+         simulationParameterOriginIdUpdater.UpdateSimulationId(_simulationToUpdate);
```

Comment:
That needed to be called after updating the simulation as new parameters do not have the reference to the simulation